app/models/pseud.rb

```diff
@@ -381,6 +382,12 @@ def check_default_pseud
     end
   end
 
+  def expire_caches
+    if name_changed?
+      self.works.each{ |work| work.touch }
```

Comment:
Redundant `self` detected.<br>Space missing to the left of {.<br>Pass `&:touch` as an argument to `each` instead of a block.